src/OpenTK/Math/Vector3.cs

```diff
@@ -257,6 +257,36 @@ public void NormalizeFast()
         public static readonly int SizeInBytes = Marshal.SizeOf(new Vector3());
 
         /// <summary>
+        /// Defines a unit-length Vector3 that points upward.
```

Comment:
OpenGL doesn't have a handedness by default :^).

@Frassle  is 100% right though, those should be defined by your framework, not ours.

I'd advise putting this kind of thing in a `Direction` static class in your code instead.